### ISSUE TITLE
Report writer UnicodeDecodeError

### ISSUE THREAD

**Issue by ddash-ct** [Defense-Cyber-Crime-Center/DC3-MWCP#31]
When attempting to add `bytes` formatted data using `metadata.DecodedString` or `metadata.Other`, a `UnicodeDecodeError` can be thrown because there is no special handling for representing the information like there is in `metadata.EncryptionKey`. This will result in a report not being generated.

An example stack trace follows:
```
Traceback (most recent call last):
  File "c:\python37\lib\site-packages\mwcp\cli.py", line 351, in parse
    print(report.as_text(format, split=split))
  File "c:\python37\lib\site-packages\mwcp\report.py", line 581, in as_text
    writer.write(self._report_model)
  File "c:\python37\lib\site-packages\mwcp\report_writers.py", line 219, in write
    self._write_table(elements)
  File "c:\python37\lib\site-packages\mwcp\report_writers.py", line 176, in _write_table
    self.table(tabular_data, headers="keys")
  File "c:\python37\lib\site-packages\mwcp\report_writers.py", line 159, in table
    self._stream.write(tabulate.tabulate(tabular_data, headers=headers, tablefmt=self._tablefmt))
  File "c:\python37\lib\site-packages\tabulate.py", line 1593, in tabulate
    for c, ct, fl_fmt, miss_v in zip(cols, coltypes, float_formats, missing_vals)
  File "c:\python37\lib\site-packages\tabulate.py", line 1593, in <listcomp>
    for c, ct, fl_fmt, miss_v in zip(cols, coltypes, float_formats, missing_vals)
  File "c:\python37\lib\site-packages\tabulate.py", line 1592, in <listcomp>
    [_format(v, ct, fl_fmt, miss_v, has_invisible) for v in c]
  File "c:\python37\lib\site-packages\tabulate.py", line 984, in _format
    return _text_type(val, "ascii")
UnicodeDecodeError: 'ascii' codec can't decode byte 0xfc in position 0: ordinal not in range(128)
Error running DC3-MWCP: 'ascii' codec can't decode byte 0xfc in position 0: ordinal not in range(128)
```

**Comment by ddash-ct** [comment]
Also attempted the following:
```
import mwcp
from mwcp import metadata
report = mwcp.Report()
with report:
	report.add(metadata.DecodedString("badstring", encryption_key=metadata.EncryptionKey(b"\xff\xff", algorithm="xor")))
print(report.as_text())
```

And received the following stack trace:
```
Traceback (most recent call last):
  File "<pyshell#27>", line 1, in <module>
    print(report.as_text())
  File "C:\Python37\lib\site-packages\mwcp\report.py", line 581, in as_text
    writer.write(self._report_model)
  File "C:\Python37\lib\site-packages\mwcp\report_writers.py", line 219, in write
    self._write_table(elements)
  File "C:\Python37\lib\site-packages\mwcp\report_writers.py", line 176, in _write_table
    self.table(tabular_data, headers="keys")
  File "C:\Python37\lib\site-packages\mwcp\report_writers.py", line 159, in table
    self._stream.write(tabulate.tabulate(tabular_data, headers=headers, tablefmt=self._tablefmt))
  File "C:\Python37\lib\site-packages\tabulate.py", line 1593, in tabulate
    for c, ct, fl_fmt, miss_v in zip(cols, coltypes, float_formats, missing_vals)
  File "C:\Python37\lib\site-packages\tabulate.py", line 1593, in <listcomp>
    for c, ct, fl_fmt, miss_v in zip(cols, coltypes, float_formats, missing_vals)
  File "C:\Python37\lib\site-packages\tabulate.py", line 1592, in <listcomp>
    [_format(v, ct, fl_fmt, miss_v, has_invisible) for v in c]
  File "C:\Python37\lib\site-packages\tabulate.py", line 984, in _format
    return _text_type(val, "ascii")
UnicodeDecodeError: 'ascii' codec can't decode byte 0xff in position 0: ordinal not in range(128)
```

**Comment by DC3-DCCI** [comment]
Thanks for catching this we hope to have a fix in the next release.

**Comment by DC3-DCCI** [comment]
This should be resolved in the 3.5.0 release.
